libs/blocks/aside/aside.js

```diff
@@ -46,32 +51,130 @@ function decorateStaticLinks(el) {
   textLinks.forEach((link) => { link.classList.add('static'); });
 }
 
+function decorateModalImage(el, lnkImg) {
+  if (el.classList.contains('split')) lnkImg.closest('div').classList.add('has-modal');
+  lnkImg.classList.add('play-btn');
+  const playIcon = createTag('div', { class: 'play-icon-container', 'aria-label': 'play' }, PLAY_ICON);
+  const playCircle = createTag('div', { class: 'play-btn-circle', 'aria-label': 'play' }, playIcon);
+  const playContainer = createTag('div', { class: 'play-container', 'aria-label': 'play' });
+  lnkImg.parentNode.appendChild(lnkImg.querySelector('picture'));
+  lnkImg.parentNode.appendChild(playContainer);
+  lnkImg.appendChild(playCircle);
+  playContainer.appendChild(lnkImg);
+}
+
+function decorateIconStack(el) {
```

Comment:
hi @hparra - I am working on support for the same icon stack feature on the `media` block and have approached this by passing this function in as a param in the decorateBlockText() func in `utils/decorate.js` [see branch](https://github.com/adobecom/milo/compare/main...rparrish/media-products)

Are you open to something like this so we can both hinge off this decorator function?

Reply:
I think you meant at someone else? I did touch decorate.js recently but it was just to autofix some eslint errors.

Reply:
@ryanmparrish Sure.. We can use a common icon block decorateIconStack for Aside and Media block. In case your PR is currently under development .. Should I go ahead and make this change in this PR so that it is available to be consumed in the Media block?

Reply:
@hparra  - Sorry, your right, This was meant for @aishwaryamathuria. 

That would be great. My task is still under dev, so I can updated after your change. Thank you

Reply:
I have made the changes suggested in PR to move the decorateIconStack to `decorate.js` so that both aside and media blocks can consume the same.
Could you please review the PR and share your comments?